Does not work with Bootstrap 4

<!--
PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.

ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
-->

## I'm submitting a...

<!-- Check one of the following options with "x" -->
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

As soon as Bootstrap 4 is added the Toast get invisible: 

https://codesandbox.io/s/musing-khayyam-83klg?file=/src/index.html

## Expected behavior

<!-- Describe what the desired behavior would be. -->

## Minimal reproduction of the problem with instructions

## What is the motivation / use case for changing the behavior?

<!-- Describe the motivation or the concrete use case. -->

## Environment

<pre><code>
Angular version: ^10
<!-- Check whether this is still an issue in the most recent Angular version -->

Browser:
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [x] Firefox version XX
- [x] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  <!-- run `node --version` -->
- Platform:  <!-- Mac, Linux, Windows -->

Others:
<!-- Anything else relevant?  Operating system version, IDE, package manager, HTTP server, ... -->

ng add does not work either:

```shell
app-test ng add @ngneat/hot-toast
Installing packages for tooling via yarn.
Installed packages for tooling via yarn.
An unhandled exception occurred: Cannot find module '@schematics/angular/utility/config'
```

and there is a undocumented dependency for @ngneat/overview

</code></pre>

--- a/projects/ngneat/hot-toast/src/lib/components/hot-toast/hot-toast.component.html
+++ b/projects/ngneat/hot-toast/src/lib/components/hot-toast/hot-toast.component.html
@@ -1,4 +1,8 @@
-<div class="hot-toast-bar-base-container" [ngStyle]="containerPositionStyle" [ngClass]="toast.theme">
+<div
+  class="hot-toast-bar-base-container"
+  [ngStyle]="containerPositionStyle"
+  [ngClass]="'hot-toast-theme-' + toast.theme"
+>
   <div
     class="hot-toast-bar-base"
     #hotToastBarBase

--- a/projects/ngneat/hot-toast/src/lib/components/hot-toast/hot-toast.component.scss
+++ b/projects/ngneat/hot-toast/src/lib/components/hot-toast/hot-toast.component.scss
@@ -41,7 +41,7 @@
     transition-duration: 10ms !important;
   }
 
-  &.snackbar {
+  &.hot-toast-theme-snackbar {
     .hot-toast-bar-base {
       color: var(--hot-toast-snackbar-color, rgba(255, 255, 255));
       background: var(--hot-toast-snackbar-bg, #323232);

